apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/ScanApp.scala

```diff
@@ -291,22 +291,45 @@ class ScanApp(
             requestLogger(traceContext) {
               HttpErrorHandler(loggerFactory)(traceContext) {
```

Comment:
Should we move the error handler below the metrics? it feels like ideally the status code produced by that is included in the metric

Reply:
moved the error handler in the route implementation, based on its implementation it should make no difference. 
I think we will still miss 404s that match no route but we should just see them with status unhandled